lexos/static/js/scripts_base.js

```diff
@@ -1,3 +1,23 @@
+$(function () {
+  if (localStorage.getItem('visited') !== 'yes') {
```

Comment:
Great work! One last thing is that.. Could you please attempt to create a function for this part and call the function in the existing: ```$(function () {})```. Add if the function works, please add a doc string for it.

Reply:
Oh yes my bad I should have done that before after testing but forgot. Just did this.